### ISSUE TITLE
Migrate to new DigitalStrom API

### ISSUE THREAD

**Issue by albertomontesg** [gaetancollaud/digitalstrom-mqtt#34]
There are any plans to migrate to the new DigitalStrom API: https://developer.digitalstrom.org/api ?

**Comment by gaetancollaud** [comment]
Holy moly, finally something new! I was waiting for it for a long time. I need to spend some time investigating but yeah for sure! I don't have time to go through it right now but I will for sure do it as soon as I have time.

Did you receive a notification or something? If yes, on which channel? Because I didn't get anything.

**Comment by albertomontesg** [comment]
Actually I encountered this surfing the web and making some searches at Google.

**Comment by gaetancollaud** [comment]
I just give it a try and it looks really really promising !
You can ask for the status of all the devices of the appartements and it returns it in ~50ms !

Documentation about auth is still really bad... Did get the bearer token to work

Websocket also works (for notifications) but seems tricky and not documented.

Note for my futur self:

```
ws://dss.local:8090/api/v1/apartment/notifications

{"protocol":"json","version":"1"}
```

**Comment by gaetancollaud** [comment]
I took some time to investigate the new API. I ended up almost reimplementing everything. I based my work on #35 to have a proper architecture.

Good news:
 - it works good for devices
 - it's fast (compare to previous implementation)
 - implementation is way simplier, no more shady work around
 - I fixed some home asisstant issues like #40 
 - Metering for the whole "apartment" (summary of the whole consumption)

Downside:
 - Stuff has been shuffled arround (for instance circtuits are now controllers, and actually they now use "meterings" for power sensors), this will be reflected in the mqtt topics
 - no more scene event (no api provided)
 - no more button event (was a workaround anyway)

I will polish it and make a new release soon, but it will be a version 2.0 and will have some breaking changes unfortunattely :tired_face:
